### PR TITLE
Update README to explain how to fill the channel

### PR DESCRIPTION
**Pull request by remimarenco**
Hi!

I have explained how to fill the channel ID in the config file. Hope it will help people who are going to use this extension!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ USERNAME = "Github"
 ICON_URL = "yourdomain.org/github.png"
 MATTERMOST_WEBHOOK_URLS = {
     'default' : ("yourdomain.org/hooks/hookid", "off-topic"),
-    'teamname/repositoryname' : ("yourdomain.org/hooks/hookid2", "repochannel"),
+    'teamname/repositoryname' : ("yourdomain.org/hooks/hookid2", "your_channel_id"),
```
Comment (ptersilie) [inline README.md]: Can we leave this as "repochannel" or better "repository-channel". I wanted to show here that you can direct notifications for a specific repository to a specific channel, e.g. the channel that is just for that repository.

I think the change below still makes sense without having to rename the channel name here.

File: README.md [hunk 2]
```diff
@@ -39,5 +39,5 @@ If you don't want to use a secret set the field to `None`.
 
 ## Known issues
 
-- Channel names need to use the spelling that is used in their URL, e.g. instead
+- Channel names need to use the spelling that is used in their URL (The channel ID), e.g. instead
```
Comment (ptersilie) [inline README.md]: Can you change the `The` to lower case? :)

**Comment by ptersilie** [comment]
Otherwise looks good. If you fix the stuff I commented we can merge this.

File: README.md (hunk 1 above)
Comment (remimarenco) [inline README.md]: :)

**Comment by remimarenco** [comment]
I think I did :)

**Comment by ptersilie** [comment]
Great. Could we also change `your_channel_id` to `repository-channel`?

**Comment by remimarenco** [comment]
Hrmm, I don't agree with this. The issue I ran into is because `repository-channel` leaded me to enter the name of my channel and not his ID.

On looked into Mattermost way of naming this field, and it seems to be <channel id> => http://forum.mattermost.org/t/api-get-latest-messages-from-a-channel/1169/2

I think this config sample is here to lead people to fill properly the fields. We should have this in mind, and that is why I pushed it like this.

I would agree on `repository-channel-id` if it fits better to your needs.

**Comment by ptersilie** [comment]
`repository-channel-id` is perfect. Thanks.

**Comment by ptersilie** [comment]
Maybe we should squash this down to a single commit and then I'm happy to merge it. :+1:

**Comment by remimarenco** [comment]
That should do it.

I would suggest not being so picky with your first commiters. A strategy that usually works is to thank people, and change it yourself the way you want. We both spent too much time on this, in my opinion.

**Comment by ptersilie** [comment]
Thanks. Merged.

**Pull request merged**